src/components/Layout/Sidebar/index.tsx

```diff
@@ -175,9 +175,11 @@ const Sidebar: React.FC<SidebarProps> = ({ open, setClosed }) => {
                   >
                     <div className="flex items-center flex-shrink-0 px-4">
                       <span className="text-xl text-gray-50">
-                        <a href="/">
-                          <img src="/logo.png" alt="Logo" />
-                        </a>
+                        <Link href="/">
+                          <a>
+                            <img src="/logo.png" alt="Logo" />
+                          </a>
+                        </Link>
```

Comment:
This was intentional to force reload the page if the logo is clicked. Helps with apps that are bookmarked on mobile devices where there is no refresh button.